mlflow/pyspark/ml/log_model_allowlist.txt

```diff
@@ -0,0 +1,37 @@
+# classification
+pyspark.ml.classification.LinearSVCModel
+pyspark.ml.classification.NaiveBayesModel
+
+# clustering
+pyspark.ml.clustering.BisectingKMeansModel
+pyspark.ml.clustering.KMeansModel
+pyspark.ml.clustering.GaussianMixtureModel
+
+# Recommendation
+pyspark.ml.recommendation.ALSModel
+
+# Regression
+pyspark.ml.regression.AFTSurvivalRegressionModel
+pyspark.ml.regression.GeneralizedLinearRegressionModel
+pyspark.ml.regression.LinearRegressionModel
```

Comment:
Can `_read_log_model_allowlist` be large?

Reply:
Won't. The model size is about `sizeof(double) * numfeatures`
logisticRegression model may be large because of numClasses may be large, it's size is about `sizeof(double) * numfeatures * numClasses`